lapack/tpls/KokkosLapack_Host_tpl.cpp

```diff
@@ -38,6 +38,27 @@ void F77_BLAS_MANGLE(cgesv, CGESV)(int*, int*, std::complex<float>*, int*, int*,
 void F77_BLAS_MANGLE(zgesv, ZGESV)(int*, int*, std::complex<double>*, int*,
                                    int*, std::complex<double>*, int*, int*);
 
+///
+/// Gesvd
+///
+
+void F77_BLAS_MANGLE(sgesvd, SGESVD)(const char*, const char*, int*, int*,
+                                     float*, int*, float*, float*, int*, float*,
+                                     int*, float*, int*, int*);
+void F77_BLAS_MANGLE(dgesvd, DGESVD)(const char*, const char*, int*, int*,
+                                     double*, int*, double*, double*, int*,
+                                     double*, int*, double*, int*, int*);
+void F77_BLAS_MANGLE(cgesvd, CGESVD)(const char*, const char*, int*, int*,
+                                     std::complex<float>*, int*, float*,
+                                     std::complex<float>*, int*,
+                                     std::complex<float>*, int*,
+                                     std::complex<float>*, int*, float*, int*);
+void F77_BLAS_MANGLE(zgesvd,
+                     ZGESVD)(const char*, const char*, int*, int*,
```

Comment:
I guess it would not hurt if we use ```const int*``` here?

Reply:
Done